src/commands/client/types.ts

```diff
@@ -1,10 +1,13 @@
 export type Vector<TMetadata = Dict> = {
   id: string;
   vector?: number[];
+  sparseVector?: SparseVector;
   metadata?: TMetadata;
   data?: string;
 };
 
 export type NAMESPACE = string;
 
 export type Dict = Record<string, unknown>;
+
+export type SparseVector = [indices: number[], values: number[]];
```

Comment:
Can we add jsdocs for this type, so that people makes sense of its structure while coding